modules/ensemble/lib/widget/input/dropdown.dart

```diff
@@ -73,7 +73,10 @@ abstract class SelectOne extends StatefulWidget
   Map<String, Function> setters() {
     var setters = _controller.textPlaceholderSetters;
     setters.addAll({
-      'value': (value) => _controller.maybeValue = value,
+      'value': (value) {
+        _controller.textEditingController.value = TextEditingValue(text: value);
```

Comment:
this should be handled internally in the widget, there is no need to expose it in the controller

Reply:
@hemish11 I am not sure why this is needed as you just override and create a new one inside initState everytime? That was my point